.github/workflows/ci.yaml

```diff
@@ -988,6 +988,12 @@ jobs:
           set -euxo pipefail
           go mod download
 
+          # The "build/coder-dylib" target depends on all Go source files and
+          # the target "agent/agentcontainers/dcspec/dcspec_gen.go" depends on
+          # node/pnpm tooling.
+          mkdir -p node_modules
+          touch node_modules/.installed
```

Comment:
/cc @deansheather lmk if you're unhappy about this change. Happy to solve this another way if you have a suggestion. I'm vary of allowing `gen/mark-fresh` to touch non-existent files (`node_modules/.installed`).

Reply:
Could you add a new `GO_SRC_FILES`-adjacent variable that excludes this directory  perhaps?

Reply:
I can do that 👍🏻. It seems the `vpn` package is limited to importing `codersdk` and `tailnet` (+ sub paths), would it be alright to limit it to these or should we have a more exhaustive selection of src files?

(It'd be nice to eventually partially generate the Makefile and/or use Go src parsing to include relevant src files for targets.)

Reply:
I think it would be fine to just exclude the problematic files for now. Even though it only imports those packages, those packages import a lot more.

Reply:
Makes sense, wdyt ef77ec5?